app/templates/gulpfile.js

```diff
@@ -50,23 +51,38 @@ function lintGulpfile() {
 }
 
 function build() {
+  var nodeModules = {};
+  fs.readdirSync('node_modules')
+    .filter(function(x) {
+      return ['.bin'].indexOf(x) === -1;
+    })
+    .forEach(function(mod) {
+      nodeModules[mod] = 'commonjs ' + mod;
+    });
   return gulp.src(path.join('src', config.entryFileName))
     .pipe(webpackStream({
+      cache: true,
+      context: __dirname,
       output: {
         filename: exportFileName + '.js',
         libraryTarget: 'umd',
         library: config.mainVarName
       },
+      debug: true,
+      target: 'node',
       // Add your own externals here. For instance,
       // {
       //   jquery: true
       // }
       // would externalize the `jquery` module.
-      externals: {},
+      externals: nodeModules,
```

Comment:
Is your goal to just externalize your node deps? If so, I'm not sure if that should be in the library out of the box. I'd be open to adding a wiki page on how to do it, though!